src/LiveDevelopment/Agents/RemoteFunctions.js

```diff
@@ -30,10 +30,29 @@
  * modules should define a single function that returns an object of all
  * exported functions.
  */
-function RemoteFunctions(experimental, remoteWSPort) {
+function RemoteFunctions(config, remoteWSPort) {
```

Comment:
I think we can pass remoteWSPort as part of config only, something like `config.remoteWSPort`.
Then instead of checking for `remoteWSPort` in https://github.com/adobe/brackets/pull/12949/files#diff-a40df952af8fd1808c321dab548ae11aR1032, we can check for `!experimental` flag as we create websocket only when we are not using experimental live preview.
But I think you can leave this for later as you might have to make changes which are not relevant for this feature, once this is merged I can make the above changes.